skt/executable.py

```diff
@@ -44,6 +45,58 @@ def full_path(path):
     return os.path.abspath(os.path.expanduser(path))
 
 
+def destroy_state_file(cfg):
+    """
+    Destroy the state file
+
+    Args:
+        cfg: A dictionary of skt configuration
+    """
+    state_file = cfg.get('state')
+    if os.path.isfile(state_file):
+        os.unlink(state_file)
+
+
+def read_state_file(cfg):
+    """
+    Read the state file from the disk
+
+    Args:
+        cfg: A dictionary of skt configuration
+    """
+    state_file = cfg.get('state')
+    if os.path.isfile(state_file):
+        with open(state_file, 'r') as fileh:
+            current_state = yaml.load(fileh)
+    else:
+        current_state = {}
```

Comment:
Can we get rid of the `current_state` variable altogether, or at least initialize it at the beginning of the function (no `else` needed)? Together with the exceptions catching we don't want to enter a code path where it would be undefined. /me remembers a similar problem with reporter